quickwit-metastore/migrations/postgresql/2021-11-10-104441_change_timestamp_fields/up.sql

```diff
@@ -0,0 +1,25 @@
+-- drop current timestamp field
+ALTER TABLE splits
+DROP COLUMN update_timestamp;
+
+-- create new timestamp fields
+ALTER TABLE splits
+ADD create_timestamp TIMESTAMP NOT NULL DEFAULT (CURRENT_TIMESTAMP AT TIME ZONE 'UTC');
+ALTER TABLE splits
+ADD update_timestamp TIMESTAMP NOT NULL DEFAULT (CURRENT_TIMESTAMP AT TIME ZONE 'UTC');
+
+-- replace diesel helper function to make sure it uses UTC timestamp
+CREATE OR REPLACE FUNCTION diesel_set_updated_at() RETURNS trigger AS $$
```

Comment:
`diesel_set_update_timestamp`?

Reply:
Yes I was hesitant on this, because we might just change the name of all the functions diesel provides for this setup

Reply:
I see. Let's not interfere with the default diesel functions and triggers then. People used to diesel might get surprises.

Let's just roll our on trigger. `quickwit_set_update_timestamp`. I'm ok with the duplication if that makes thing more explicit/transparent.